src/materialized/tests/sql.rs

```diff
@@ -785,7 +785,7 @@ fn test_tail_shutdown() -> Result<(), Box<dyn Error>> {
         conn_task.abort();
 
         // Need to await `conn_task` to actually deliver the `abort`. We don't
-        // care about the result though (it's probably `JoinError::Cancelled`).
+        // care about the result though (it's probably `JoinError::Canceled`).
```

Comment:
Unfortunately this one comes from upstream where they use the "ll" spelling. The actual variant is gone but the `is_cancelled` method still exists: https://docs.rs/tokio/latest/tokio/task/struct.JoinError.html#method.is_cancelled.